.github/workflows/build.yml

```diff
@@ -42,5 +43,5 @@ jobs:
         working-directory: packages/${{ matrix.package }}
 
       - name: Upload coverage to codecov
-        run: curl -s https://codecov.io/bash | bash
+        run: curl -s https://codecov.io/bash | bash -s -- -s coverage
```

Comment:
What is the purpose of this?

Reply:
The purpose is to tell codecov to search for coverage directly in coverage folder, like this both package can provide their coverage and codecov is fusioning both